Layout/UIView+Layout.swift

```diff
@@ -278,6 +278,19 @@ extension UIView: LayoutManaged {
         }
     }
 
+	// Set expression value from string
+	@objc open func setStringValue(_ value: String, forExpression name: String) throws {
+		if let typ = type(of: self).cachedExpressionTypes[name] {
+			if let node = _layoutNode {
+				if let exp = LayoutExpression(expression: value, type: typ, for:node) {
```

Comment:
These `if`s should all be combined into a single `guard` statement to avoid the "pyramid of doom".

Reply:
Sure